src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -3998,4 +3998,7 @@ Activate by holding the key for the character you want to add an accent to, then
   <data name="PowerLauncher_TextFontSizeSlider.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
     <value>Text size of result titles</value>
   </data>
+  <data name="CmdNotFound_Arm64ArchBar.Title" xml:space="preserve">
+    <value>Command Not Found is not support on ARM64 architecture.</value>
```

Comment:
```suggestion
    <value>Currently, Command Not Found is not supported on the ARM64 architecture, since there's no official ARM64 .msi release of PowerShell 7.4.</value>
```